src/test/regress/sql/propagate_set_commands.sql

```diff
@@ -13,10 +13,53 @@ SET citus.select_opens_transaction_block TO on;
 
 BEGIN;
 SELECT current_setting('enable_hashagg') FROM test WHERE id = 1;
--- should not be propagated, error should be coming from coordinator
+-- triggers an error on the worker
 SET LOCAL TRANSACTION ISOLATION LEVEL REPEATABLE READ;
 END;
 
+BEGIN;
```

Comment:
probably not needed as test, but just sharing a serializable tx error, which is handled fine:
```SQL
CREATE TABLE test(a int, b int);
SELECT create_distributed_table('test', 'a');

INSERT INTO test VALUES (1,1);

SET citus.propagate_set_commands TO 'local';

-- session 1
BEGIN;

SET TRANSACTION ISOLATION LEVEL SERIALIZABLE;
UPDATE test SET b = 2 WHERE a = 1;


-- session 2
BEGIN;

SET TRANSACTION ISOLATION LEVEL SERIALIZABLE;
SELECT * FROM test;

-- session 1 
COMMIT;

-- session 2
UPDATE test SET b = 2 WHERE a = 1;
ERROR:  could not serialize access due to concurrent update
CONTEXT:  while executing command on localhost:9701
```